homework.rb

```diff
@@ -1,6 +1,10 @@
-VARIANT = 0
+VARIANT = 8
 
 def generate
-  # put your solution here (delete the line below)
-  rand(1000000).to_s + '-' + rand(1000000).to_s
+  letters = *("A".."Z")
+
+  letter1, letter2 = 2.times.map { letters.sample(3).join }
```

Comment:
Yes, And Lambda gives you runtime calls. Lambda/Proc/Block are very cool and widely used construction. Please, pay attention to learn them good enough.